core/templates/pages/exploration-player-page/learner-experience/ratings-and-recommendations.component.ts

```diff
@@ -55,26 +59,50 @@ export class RatingsAndRecommendationsComponent {
   @Input() userIsLoggedIn: boolean;
   @Input() explorationIsInPreviewMode: boolean;
   @Input() questionPlayerConfig: QuestionPlayerConfig;
-  @Input() inStoryMode: boolean;
-  @Input() storyViewerUrl!: string;
   @Input() collectionSummary: CollectionSummary;
   @Input() isRefresherExploration: boolean;
   @Input() recommendedExplorationSummaries: LearnerExplorationSummary[];
   @Input() parentExplorationIds: string[];
+  inStoryMode: boolean;
+  storyViewerUrl!: string;
```

Comment:
Sorry, this was a mistake. storyViewerUrl will only be defined if the exploration is being played on story mode, i.e. if inStoryMode is true (it will remain undefined otherwise).